db/migrate/033_create_job_definition_revisions.rb

```diff
@@ -0,0 +1,21 @@
+class CreateJobDefinitionRevisions < ActiveRecord::Migration[5.0]
+  def change
+    remove_column :job_definitions, :version
+    remove_column :job_instances, :job_definition_version
+    remove_column :tokens, :job_definition_version
+    remove_column :executions, :job_definition_version
+
+    create_table :job_definition_revisions do |t|
+      t.belongs_to :job_definition, foreign_key: true, null: false
+      t.text :script, null: false
+      t.belongs_to :user, foreign_key: true, null: true
+      t.datetime :changed_at, null: true
+
+      t.timestamps null: false
+    end
+
+    Kuroko2::JobDefinition.all.each do |definition|
+      Kuroko2::JobDefinitionRevision.create(job_definition: definition, script: definition.script)
```

Comment:
This migration should not be run in `rails db:rollback`

Reply:
Thanks. I fixed in 8cc9ec470334c210afd46d2db4358a5de5725cc8.